src/Symfony/Bundle/FrameworkBundle/DependencyInjection/Configuration.php

```diff
@@ -406,7 +406,7 @@ private function addWorkflowSection(ArrayNodeDefinition $rootNode): void
                                                 ->values(['method'])
                                             ->end()
                                             ->scalarNode('property')
-                                                ->defaultValue('marking')
```

Comment:
This is a BC break?
And is was nice to have a default value...

Reply:
see https://github.com/symfony/symfony/pull/51148/files#diff-e4f52d5033296421d02e360c8355aa1931bdfb0c12579f4815ea02b724322c0cR1005

Reply:
oups, I confused `cannotBeEmpty` and `isRequired`